src/js/components/atoms/buttons/anchor/data/examples/anchor-button.data.ts

```diff
@@ -0,0 +1,16 @@
+export const data = {
```

Comment:
Why ```examples``` folder? Are there files in the data folder as-well?

Reply:
when i created card-product i noticed that i had several data possibilities, so i put them in separate files in an examples folder, but you have a good point; after the re-factoring i did here it no longer seems necessary to separate the files out this way.

i’ll get rid of the examples folder and move the knobs file directly into the stories folder.